stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/controller/ChannelControllerImpl.kt

```diff
@@ -435,59 +435,38 @@ internal class ChannelControllerImpl(
         val queryOfflineJob = domainImpl.scope.async { runChannelQueryOffline(pagination) }
 
         // start the online query before queryOfflineJob.await
-        val queryOnlineJob = if (domainImpl.isOnline()) {
-            domainImpl.scope.async { runChannelQueryOnline(pagination) }
-        } else {
-            // if we are not offline we mark it as needing recovery
-            recoveryNeeded = true
```

Comment:
We lose this line both here and in the other controller with these changes. Do we not need to set this flag when we're offline?

Reply:
This flag is updated into `runChannelQueryOnline()` when the query fails, so it is not needed anymore here